aclogview/CM_Writing.cs

```diff
@@ -55,30 +55,30 @@ public class CM_Writing : MessageProcessor {
     }
 
     public class BookData : Message {
-        public uint i_objectID;
+        public uint i_objectid;
 
         public static BookData read(BinaryReader binaryReader) {
             BookData newObj = new BookData();
-            newObj.i_objectID = binaryReader.ReadUInt32();
+            newObj.i_objectid = binaryReader.ReadUInt32();
             return newObj;
         }
 
         public override void contributeToTreeView(TreeView treeView) {
             TreeNode rootNode = new TreeNode(this.GetType().Name);
             rootNode.Expand();
-            rootNode.Nodes.Add("i_objectID = " + i_objectID);
+            rootNode.Nodes.Add("i_objectid = " + i_objectid.ToString("X"));
```

Comment:
Any reason that a few of these always use hex and don't check to see if UseHex is enabled?